docs/source-app/glossary/storage/drive_content.rst

```diff
@@ -4,13 +4,24 @@
 About Drives
 ************
 
-Lightning Drive storage makes it easy to share files between LightningWorks so you can run your Lightning App both locally and in the cloud without changing the code.
+Lightning Drive storage makes it easy to share files between LightningWorks (Work) so you can run your Lightning App both locally and in the cloud without changing the code.
 
 The Drive object provides a central place for your components to share data.
 
-The Drive acts as an isolate folder and any component can access it by knowing its name.
+The Drive acts as an isolated folder and any component can access it by knowing its name.
 
-Your components can put, list, get, and delete files from and to the Drive (except LightningFlows).
+We currently support two types of Drives: ``lit://`` and ``s3://``.
```

Comment:
We should name the types, and then in the table explain how to create each.
```suggestion
We currently support two types of Lighting Drives: Lightning-managed and S3.
```

Reply:
I like this, let’s do “Lightning-managed (``lit://``) and S3 (``s3://``)?